### ISSUE TITLE
CQ ,ITU

### ISSUE THREAD

**Issue by ra4lp** [VK6NX/OCLog-v1.x#5]
Не правильно выдает информацию на часть российских позывных CQ ,ITU. Например UA4L cq15 должно быть 16. UA4H cq15     
itu 30 должно быть 16/29.
Incorrectly issues information on part of the Russian callsigns CQ, ITU. For example UA4L cq 15 should be 16. UA4H cq 15 itu 30 should be 16/29.

**Comment by VK6NX** [comment]
Да, ошибка... Буду проверять весь файл целиком, видимо при конвертации сбой был.

Поправлю.

**Comment by ra4lp** [comment]
OK! 73!

**Comment by VK6NX** [comment]
поправил большую часть префиксов.

Насчет UA4H уточнение:
тут https://radiostorage.net/3613-pozyvnye-radiostancij-rossii.html и тут https://zone-check.eu/ UA4H выдается как cq 16 itu 30 (Самара и Самарская обл как исключение из общего списка 16 29 для UA4H UA4I). Так что пока что оставлю их 16 30.

В любом случае поправить ITU CQ и DXCC можно через меню Settings -> DXCC - поиск по префиксу - Edit
А вернуть полный лист к первоначальному можно нажав кнопку default.

Версия с новым DXCC листом будет выложена либо сегодня позже, либо завтра.

**Comment by ra4lp** [comment]
Хорошо!28.10.2022, 08:57, "VK6NX" ***@***.***>:
поправил большую часть префиксов.
Насчет UA4H уточнение:
тут https://radiostorage.net/3613-pozyvnye-radiostancij-rossii.html и тут https://zone-check.eu/ UA4H выдается как cq 16 itu 30 (Самара и Самарская обл как исключение из общего списка 16 29 для UA4H UA4I). Так что пока что оставлю их 16 30.
В любом случае поправить ITU CQ и DXCC можно через меню Settings -> DXCC - поиск по префиксу - Edit
А вернуть полный лист к первоначальному можно нажав кнопку default.
Версия с новым DXCC листом будет выложена либо сегодня позже, либо завтра.

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>
-- Alex Z.

**Comment by VK6NX** [comment]
update 2 опубликован, должно работать

**Comment by ra4lp** [comment]
Работает но только для UA4L , короткие позывные такие как R4L, R4M CQ, ITU не показывает. UA3,UA4 остальные остались в CQ15 должно быть СQ16.

**Comment by VK6NX** [comment]
Надо сделать принудительную перезапись базы данных. Она однаковая по ID с предыдущей версией, поэтому данные не перезаписала.

settings->DXCC и нажать кнопку Default (и подтвердить Load).

С короткими позывными я пока не разбирался, они находят совпадение по базе по R - а у нее все данные = "точки". Если структура коротких позывных такова, что R в позывном R4L можно прираянять в RA4L - то это легко будет сделать.

**Comment by ra4lp** [comment]
Спасибо ! Перезаписал ,буду пробовать дальше  .

**Comment by ra4lp** [comment]
Подскажите как дела с CWSkimmer для SunSDR ???

**Comment by VK6NX** [comment]
> Подскажите как дела с CWSkimmer для SunSDR ???

Вышли из праздничных загулов :), приступили. Планируем до середины месяца выдать что-то приемлемое, но не уверен, что сразу будет под все ОС. Скорее всего начнем с версии для маков.

**Comment by ra4lp** [comment]
Здравствуйте! Установил обновление UPDATE4.При нажатии на CONNECT журнал не стыкуется с SunSdr2PRO,
пока не нажмёшь CW Macro. Или при запуске нужно сначала нажать CW MACRO, затем CONNECT тогда есть подключение.
В ранних версиях всё нормально, при нажатии на Connect журнал сразу подключается к трансиверу.

**Comment by VK6NX** [comment]
Разницы базового функционала лога между u3 и u4 нет. Так что можно использоваться и одну и другую версию.

В u4 добавлен протокол обмена для скиммера. Соответственно, в QSLite базе данных установок появилась новая колонка: wsSkimmerIPport. Ну а поскольку в ОС уже есть базе данных установок от версии  u3, то u4  пытается ей воспользоваться. Но "упирается" в разный размер табличек. Что нужно сделать, чтобы заработало с u4 - записать все свои установки станций в скриншот или куда-то еще, затем руками удалить базу (в описании есть данные где она находится), после удаления запустить u4 и снова завести все установки станций.

Ну или можно просто вернуться на u3. Обмен со скиммером добавлен для тестирования, и  он будет апдейтиться еще пару раз, поскольку в скиммере мы наткнулись на пару моментов и сейчас их решаем.

**Comment by ra4lp** [comment]
Понятно, Спасибо за разъяснения, буду пробовать.05.03.2023, 05:13, "VK6NX" ***@***.***>:
Разницы базового функционала лога между u3 и u4 нет. Так что можно использоваться и одну и другую версию.
В u4 добавлен протокол обмена для скиммера. Соответственно, в QSLite базе данных установок появилась новая колонка: wsSkimmerIPport. Ну а поскольку в ОС уже есть базе данных установок от версии  u3, то u4  пытается ей воспользоваться. Но "упирается" в разный размер табличек. Что нужно сделать, чтобы заработало с u4 - записать все свои установки станций в скриншот или куда-то еще, затем руками удалить базу (в описании есть данные где она находится), после удаления запустить u4 и снова завести все установки станций.
Ну или можно просто вернуться на u3. Обмен со скиммером добавлен для тестирования, и  он будет апдейтиться еще пару раз, поскольку в скиммере мы наткнулись на пару моментов и сейчас их решаем.

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>
-- Alex Z.
